### ISSUE TITLE
Setup Button Navigation between All Scenes

### ISSUE THREAD

**Issue by SilasReinagel** [EnigmaDragons/ProjectNeon#59]
Add all game scenes to the build.
Handle navigation via Events.

Use methods or string mapping for navigation, not numeric references.

Blocked by #11 and #12.

**Comment by 0crat** [comment]
Job was finished in 39 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#59](https://github.com/enigmadragons/projectneon/issues/59) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)
